guides/Release_Notes/master.adoc

```diff
@@ -0,0 +1,29 @@
+include::common/attributes.adoc[]
+
+:imagesdir: common/images
+ifdef::foreman-el,foreman-deb,katello[]
+:toc:
+:toc-placement: left
+endif::[]
+
+:upstream-release-notes:
+:context: {project-context}
+:ProductName: {ProjectServer}
+
+ifdef::foreman-el[]
+include::common/modules/ref_foreman-3-0-release-notes.adoc[]
+
+## Corresponding Katello release notes
+
+
+include::common/modules/ref_katello-release-notes.adoc[]
```

Comment:
I don't fully understand the logic behind this. Is it mixed up? Only show katello release notes "ifdef katello"?